feature-libs/cart/quick-order/components/quick-order/form/quick-order-form.component.ts

```diff
@@ -68,6 +79,7 @@ export class QuickOrderFormComponent implements OnInit, OnDestroy {
     form.setControl('product', new FormControl(null));
 
     this.form = form;
+    this.validateProductControl(this._disabled);
```

Comment:
```suggestion
    this.validateProductControl(this.isDisabled);
```

this your using getters/setters to define `this._disabled`